javascripts/discourse/initializers/category-icons.js

```diff
@@ -16,7 +16,7 @@ export default {
   name: "category-icons",
 
   initialize() {
-    withPluginApi("0.8.26", (api) => {
+    withPluginApi("1.6.1", (api) => {
```

Comment:
With the conditional below, I don't think you need to change the version number here. I believe changing this actually means that people on Discourse version before 1.6.1 who update this theme component will no longer see their icons anywhere. 

I believe just the conditional below is probably enough.